sdk-tests/components/statestore.yaml

```diff
@@ -10,5 +10,3 @@ spec:
     value: localhost:6379
   - name: redisPassword
     value: ""
-  - name: actorStateStore
```

Comment:
I had to change the actor state store, so the work left is to use mysql and add all actor apps to the scopes of the new mysql state store.